backend/pkg/connect/guides/s3_sink_connector.json

```diff
@@ -57,26 +57,35 @@
       "definition": {
         "display_name": "AWS S3 region",
         "group" : "Amazon S3",
+        "importance": "HIGH",
         "order": 4
       }
     },
+    {
+      "nameSelector": "aws.s3.bucket.check",
+      "definition": {
+        "display_name": "AWS S3 bucket check",
+        "group" : "Amazon S3",
+        "order": 5
+      }
+    },
```

Comment:
@tomasz-sadura Is that an option that is provided by the targetted sink connector?

Reply:
yes